x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/sample_logs_input.test.tsx

```diff
@@ -174,7 +174,7 @@ describe('SampleLogsInput', () => {
 
         it('should truncate the logs sample', () => {
           expect(mockActions.setIntegrationSettings).toBeCalledWith({
-            logSamples: tooLargeLogsSample.split(',').slice(0, 10),
+            logSamples: tooLargeLogsSample.split(',').slice(0, 2),
```

Comment:
I am not sure how this works, but apparently `expect...toBeCalledWith(['A', 'B'])` here accepts being called with something like ['B', 'A', 'A', 'B', 'A'].

Reply:
I don't think this example would pass, are we sure we are cleaning all teh calls from previous test cases?